src/ha_addon_sunsynk_multi/sensor_options.py

```diff
@@ -80,18 +107,7 @@ def init_sensors(self) -> None:
                     visible=True,
                     first=True,
                 )
-
-        # Handle RW sensor deps
-        for sopt in list(self.values()):
-            if isinstance(sopt.sensor, RWSensor):
-                for dep in sopt.sensor.dependencies:
-                    self.startup.add(dep)
-                    if dep not in self:
-                        self[dep] = SensorOption(
-                            sensor=dep,
-                            schedule=get_schedule(dep, SCHEDULES),
-                        )
-                    self[dep].affects.add(sopt.sensor)
+                self._add_sensor_with_deps(sen, visible=True)
```

Comment:
Don't we need a first=True here as well? (passed to the SensorOptions?) - #374 made me think of this